.vscode/cspell.json

```diff
@@ -233,6 +233,7 @@
     "myfile",
     "nazsdk",
     "nbytes",
+    "nbsp",
```

Comment:
What's this for?

Reply:
It is used in the html page:

https://github.com/Azure/azure-sdk-for-python/blob/core-experimental/sdk/core/azure-core-experimental/samples/pyodide_integration/index.html

cspell was not happy. :)